### PR TITLE
adds Atom instructions, and fake file for visibility 

### PR DESCRIPTION
**Pull request by janat08**

### CODE REVIEW

**Comment by janat08** [comment]
#13

**Comment by veered** [comment]
Thanks! I'll take a look this weekend

File: plugins/atom.js.readBelow [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (veered) [inline plugins/atom.js.readBelow]: What is the purpose of this file?

File: plugins/README.md [hunk 2]
```diff
@@ -20,3 +20,27 @@ Go to File -> Settings -> Plugins -> Install from disk and select
 reval.jar. You will need to restart the IDE to see changes. By default
 the keymap shortcut is Ctrl + Alt + S, but you can change it on
 File -> Settings -> Keymap -> Main menu -> Code -> Update Blaze Code
+
+
+### Atom
+Go to Edit -> [Init Script..., Keymaps...]
+In init.coffee paste following (with notion that indentation is important for script):
+```
+http = require('http')
```
Comment (veered) [inline plugins/README.md]: It would be more consistent w/ the other plugins if the actual plugin code was in its own file.

**Comment by veered** [comment]
I added a couple comments

File: plugins/atom.js.readBelow (hunk 1 above)
Comment (janat08) [inline plugins/atom.js.readBelow]: fake file since it mentions nowhere that atom is supported like a title.

File: plugins/README.md (hunk 2 above)
Comment (janat08) [inline plugins/README.md]: atom is unique, this file would have nowhere to go unless an actual package, and if using key based patching it's even misleading.

**Comment by veered** [comment]
Closing in favor of https://github.com/qualialabs/reval/pull/21

**Pull request closed without merging**